### PR TITLE
Localization ReactPlay In Bengali

### PR DESCRIPTION
**Pull request by Nargis21**
### **Issue no: #60** 
As a user of ReactPlay, I would like to suggest the localization of the website in the Bengali language. By providing content in the native language of Bangla-speaking users, ReactPlay can greatly enhance the user experience. Users will feel more comfortable navigating the website, understanding instructions, and accessing features without any language barriers. It will make the website more accessible and user-friendly, attracting a wider audience. I completed the localization in Bengali.

### CODE REVIEW

**Comment by uiSubho98** [comment]
Subhojit Dutta

**Comment by atapas** [comment]
@uiSubho98 

Thanks for volunteering to review it.

Please go page by page using the preview link above.

Do these pages first:

- [ ] https://docs.reactplay.io/
- [ ] https://docs.reactplay.io/contribution-guide
- [ ] https://docs.reactplay.io/local-development

If you have the review comments, please post them on the PR itself. I have added you ad the reviewer now.

**Review by priyankarpal — CHANGES_REQUESTED** [review]
please fix the spelling 
image

**Comment by Nargis21** [comment]
Hey @priyankarpal, Is it okay? actually, it looks like this from my side,
image

**Comment by priyankarpal** [comment]
> Hey @priyankarpal, Is it okay? actually, it looks like this from my side, but don't know why the letters broken from your end. image

yeah! now it's ok

File: i18n/bn/docusaurus-plugin-content-docs/current/local-development.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+npm install --legacy-peer-deps
+```
+
+## আপনার প্রজেক্ট ফোল্ডারের রুটে একটি .env ফাইল তৈরি করুন এবং নিম্নলিখিত কনটেন্ট লিখুন।
+
+```bash
+REACT_APP_NHOST_BACKEND_URL=https://rgkjmwftqtbpayoyolwh.nhost.run/
+REACT_APP_NHOST_VERSION=v1
+REACT_APP_NHOST_ENDPOINT=graphql
+```
```
Comment (priyankarpal) [inline i18n/bn/docusaurus-plugin-content-docs/current/local-development.md:74]:
```suggestion
```bash

REACT_APP_PLAY_API_URL=https://api.github.com/repos/reactplay
REACT_APP_NHOST_BACKEND_URL=https://rgkjmwftqtbpayoyolwh.nhost.run
REACT_APP_NHOST_VERSION=v1
REACT_APP_NHOST_ENDPOINT=graphql
REACT_APP_PLAY_WEB_SVC=https://api.reactplay.io/.netlify/functions/server
DISABLE_ESLINT_PLUGIN=true
REACT_APP_ACTIVITIES_ON=true
REACT_APP_ACTIVITY_ID=hackrplay
REACT_APP_DADJOKES_URL=https://jokeapi-v2.p.rapidapi.com/joke/
REACT_APP_DADJOKES_APIKEY='b71df95c75msha446fab91d0e935p1d0262jsn1d938cb85502'
REACT_APP_DADJOKES_APIHOST='jokeapi-v2.p.rapidapi.com'

```

**Review by priyankarpal — CHANGES_REQUESTED** [review]
@Nargis21 please check

File: i18n/bn/docusaurus-plugin-content-docs/current/local-development.md (hunk 1 above)
Comment (Nargis21) [reply i18n/bn/docusaurus-plugin-content-docs/current/local-development.md:74]: @priyankarpal Done!

Comment (Nargis21) [reply i18n/bn/docusaurus-plugin-content-docs/current/local-development.md]: @priyankarpal Done!

**Review by priyankarpal — APPROVED** [review]

**Review by atapas — APPROVED** [review]

**Pull request merged**